observability/otel/otel-collector/otel-collector-processors.adoc

```diff
@@ -453,3 +453,108 @@ processors:
 ----
 <1> Specifies attribute keys to group by.
 <2> If a processed span, log record, or metric datapoint contains at least one of the specified attribute keys, it is reassigned to a Resource that shares the same attribute values; and if no such Resource exists, a new one is created. If none of the specified attribute keys is present in the processed span, log record, or metric datapoint, then it remains associated with its current Resource. Multiple instances of the same Resource are consolidated.
+
+[id="transform-processor_{context}"]
+== Transform Processor
+
+The Transform Processor enables modification of telemetry data according to specified rules and in the link:https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/ottl[OpenTelemetry Transformation Language (OTTL)].
+For each signal type, the processor processes a series of conditions and statements associated with a specific OTTL Context type and then executes them in sequence on incoming telemetry data as specified in the configuration.
```

Comment:
Seems like this could be lowercase, or perhaps it should be `context` type like in line 466?

```suggestion
For each signal type, the processor processes a series of conditions and statements associated with a specific OTTL context type and then executes them in sequence on incoming telemetry data as specified in the configuration.
```